### PR TITLE
chore: add classifier for python 3.10

### PR DESCRIPTION
**Pull request by anukaal**
added support for python 3.10

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [3.6, 3.7, 3.8, 3.9]
+        python: [3.6, 3.7, 3.8, 3.9, 3.10]
```
Comment (software-dov) [inline .github/workflows/tests.yml:49]: The really horrible annoying fact is that github actions doesn't like `3.10`: it needs to be in quotes.

**Comment by software-dov** [comment]
Hi, thanks for contributing! Be sure to tweak the title of your PR to use [conventional commits](https://www.conventionalcommits.org/) before submitting.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (anukaal) [reply .github/workflows/tests.yml:49]: > The really horrible annoying fact is that github actions doesn't like `3.10`: it needs to be in quotes.

I guess all the version should be in quotes that would be better

Comment (software-dov) [reply .github/workflows/tests.yml:49]: That sounds good for consistency. My guess is that they're being interpreted as floating point numbers, which means stripping trailing zeros.

Comment (anukaal) [reply .github/workflows/tests.yml:49]: Yeah , it would be great if all the python version should be in quotes

**Review by parthea — APPROVED** [review]

**Pull request merged**